src/styles/getTooltipStyles.js

```diff
@@ -111,12 +114,13 @@ export default function getTooltipStyles(
     tooltipContentWidth,
     manualShiftHorizontal = 0,
     manualShiftVertical = 0,
+    wrapper,
```

Comment:
"Wrapper" is the term used in the nearby code for the wrapper element of the rendered tooltip. See the only usage of this function. It's the thing used to "Ensure it's not itself".

Reply:
I mean [this guy](https://github.com/Expensify/App/blob/b51738279a4f11e5957c9595443c4dea303627ee/src/components/Tooltip/TooltipRenderedOnPageBody.js#L70).

Reply:
Maybe this thing should actually be called "tooltip root"? As the "tooltip wrapper" is indeed used when talking about the wrapper width/height passed from `Tooltip`, which is a different thing.

So I could rename the `wrapper` variable in `TooltipRenderedOnPageBody` to `root` and refer to it as "tooltip's root". Okey?

Reply:
Okey, so I approached it in a slightly different way. I left the "wrapper" nomenclature, but I introduced a new term, "tooltip target", which is the thing that the `Tooltip` class describes. Then I used the term "target" when referring to the target and "tooltip" when referring to the actual tooltip, which points at the target. Both have a wrapper, i.e. an element that wraps their content.

I also dropped "tooltip" prefix in a few places, because I felt it was used inconsistently, and partially to separate the tooltip from the tooltip target.

See the new commit. If you disagree with this approach, we can partially/fully revert it.